Fabric.Authorization.API/Modules/RootModule.cs

```diff
@@ -1,19 +1,24 @@
-﻿using Nancy;
+﻿using Fabric.Authorization.API.Configuration;
+using Nancy;
 using Fabric.Authorization.API.Constants;
+using Fabric.Authorization.Domain.Models;
 
 namespace Fabric.Authorization.API.Modules
 {
     public class RootModule : NancyModule
     {
-        public RootModule(): base("/v1")
+        private readonly IAppConfiguration _appConfiguration;
+        public RootModule(IAppConfiguration appConfiguration): base("/v1")
         {
+            _appConfiguration = appConfiguration;
             Get("/", _ => Redirect(), null, "Redirect");
             Get($"/{AccessControl.Path}", _ => Redirect(), null, "Redirect");
         }
 
         private dynamic Redirect()
         {
```

Comment:
Can we use `IHostingEnvironment.WebRootPath` here to avoid the need for a new configuration variable (similar to what I did in https://github.com/HealthCatalyst/Fabric.Authorization/blob/27af629a9b98963380a4b1e7e6205585c5fffc55/Fabric.Authorization.API/Infrastructure/Middleware/AngularMiddleware.cs)?

Reply:
I'll test that out and see.

Reply:
WebRootPath doesn't provide a URL, it provides a file path to the web content, in this case we are redirecting to a URL, so we can't take advantage of the file path.

Reply:
@michaelvidal24, that's right. Sorry about that.